Identity/src/Migrations/201601280309540_UpdatedUser.cs

```diff
@@ -7,10 +7,10 @@ public partial class UpdatedUser : DbMigration
         public override void Up()
         {
             AlterColumn("dbo.Users", "Active", c => c.String());
-
-            base.Sql("UPDATE dbo.Users SET CASE WHEN Active = 1 THEN 'active' ELSE 'inactive'");
+            
+            base.Sql("UPDATE dbo.Users SET Active = CASE WHEN Active = 1 THEN 'active' ELSE 'inactive' END");
```

Comment:
Yes, that would help :smile: 

I wrote that late at night after manually updating my database. And obviously hadn't tested it yet.